cmd/chart-repo/sync.go

```diff
@@ -60,7 +60,8 @@ var syncCmd = &cobra.Command{
 			logrus.Fatalf("Can't connect to mongoDB: %v", err)
 		}
 
-		if err = syncRepo(dbSession, args[0], args[1]); err != nil {
+		accessToken := os.Getenv("ACCESS_TOKEN")
```

Comment:
I think we should rename this to `AUTHORIZATION_HEADER` to make it more clear?

Reply:
done!